### PR TITLE
feature: add community templates link to templates docs (#2339).

### LINKED ISSUE #2339: Request to add link to templates on InfluxDB templates doc
_Describe your proposal here._
It would be great to have a link to our [community templates](https://github.com/influxdata/community-templates/) on [this doc](https://docs.influxdata.com/influxdb/v2.0/influxdb-templates/#template-resources).

##### Relevant URLs
https://docs.influxdata.com/influxdb/v2.0/influxdb-templates/#template-resources

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #2339

### CODE REVIEW

File: content/influxdb/v2.0/influxdb-templates/_index.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ share your configuration with the InfluxData community.
 
 {{< youtube 2JjW4Rym9XE >}}
 
+[InfluxData Community Templates](https://github.com/influxdata/community-templates/)
```
Comment (sanderson) [inline content/influxdb/v2.0/influxdb-templates/_index.md:25]: This is a pretty significant feature for new users, so I'm wondering if we bump up the hierarchy of this link by formatting it as a button and a call to action.

```suggestion
<a class="btn github" href="https://github.com/influxdata/community-templates/" target="_blank">View InfluxDB community templates</a>
```

Side note – I need to create a shortcode for creating buttons 😄

**Review by sanderson — APPROVED** [review]
Looks good. Just one minor thought.

**Comment by jstirnaman** [comment]
Thanks! I was thinking the same thing, but wasn't sure whether it qualified.

File: content/influxdb/v2.0/influxdb-templates/_index.md (hunk 1 above)
Comment (jstirnaman) [reply content/influxdb/v2.0/influxdb-templates/_index.md:25]: Thanks! I was thinking the same thing, but wasn't sure whether it qualified.

**Pull request merged**